src/Simulation/QCTraceSimulator/SortedQubitPool.cs

```diff
@@ -0,0 +1,329 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+
+namespace Microsoft.Quantum.Simulation.QCTraceSimulatorRuntime {
+
+    /// <summary>
+    /// Either time interval or point in time on the timeline of the program.
```

Comment:
Don't understand: how this class can represent an interval?

Reply:
I updated the comment. It's the length of a time interval (not the interval itself). And it's the same way as number 5 can represent duration or a point in time.